doc/installation/linux_install.rst

```diff
@@ -71,15 +71,22 @@ The following are the basic steps to compile and install NEST from source code:
 
 NEST should now be successfully installed on your system. You should now be able to ``import nest``  from a python or ipython shell.
 
-.. admonition:: IMPORTANT!
+Setting the module loading path
+-------------------------------
 
- If your operating system does not find the ``nest`` executable or if Python does not find the ``nest`` module, your path variables may not be set correctly. This may also be the case if Python cannot load the ``nest`` module due to missing or incompatible libraries. In this case, please run:
+If your operating system does not find the ``nest`` executable or if Python does not find the ``nest`` module, your path variables may not be set correctly. This may also be the case if Python cannot load the ``nest`` module due to missing or incompatible libraries.
+
+In order to correctly set the module loading path or build an extension module for NEST:
+
+1. Source ``nest_vars.sh``:
 
   .. code-block:: sh
 
-       source </path/to/nest_install_dir>/bin/nest_vars.sh
+      source </path/to/nest_install_dir>/bin/nest_vars.sh
+
+2. Set the ``NEST_MODULE_PATH``.
 
```

Comment:
I would include a line how to actually do it:
```suggestion
2. Set the ``NEST_MODULE_PATH``.

  .. code-block:: sh
      export NEST_MODULE_PATH=</path/to/your/modules/>
      
```

Reply:
@terhorstd This file got updated after the merge of sarakonradi/nest-simulator#2. Please check again!